lib/surrounded/access_control.rb

```diff
@@ -6,16 +6,16 @@ def self.extended(base)
         base.const_set(:AccessError, Class.new(::Surrounded::Context::AccessError))
       end
     end
-    
```

Comment:
I've moved the project over to my Cloud9 account, which puts it on a linux box. I'll see how this evening's commit goes.